crud.controller.js

```diff
@@ -573,6 +587,8 @@ CrudController.prototype = {
         var upsert = reqParams['upsert'];
         var docIds = [];
         var body = params.map(req)[payload];
+        var restrictOnError = reqParams['restrictOnError'] && Array.isArray(body);
```

Comment:
The param is not `restrictOnError`. It is `batchMode`

Reply:
Hey, you told to make it abortOnError right?